Description: Do not build the module info (until the dependencies have their own, especially junit-jupiter-api)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/pom.xml
+++ b/pom.xml
@@ -435,9 +435,6 @@
   <profiles>
     <profile>
       <id>java9+</id>
-      <activation>
-        <jdk>[9,)</jdk>
-      </activation>
       <properties>
         <!-- allow tests to access private fields/methods of java.base classes via reflection -->
         <argLine>--add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED
